Lib/test/test_uuid.py

```diff
@@ -527,21 +529,22 @@ def check_node(self, node, requires=None, *, random=False):
         # `test_random_getnode()` method specifically.  Another case is the
         # Travis-CI case, which apparently only has locally administered MAC
         # addresses.
-        if not random and not os.getenv('TRAVIS'):
+        if not local and not os.getenv('TRAVIS'):
```

Comment:
We shouldn't test the multicast bit for `_unix_getnode()` and `_windll_getnode()`.

Reply:
Good point.  I'm removing the multicast bit check from `check_node()` since only the test for `_random_getnode()` needs to check that.